README.md

```diff
@@ -1,28 +1,130 @@
+included for x86 and ARM.
+
+## Quickstart for Amazon Linux 2
+Fork AWS-LC on GitHub and run the following commands to build AWS-LC with optimizations
+and debug info, run all tests, and install it:
+```bash
+sudo yum install cmake ninja-build clang perl golang
+git clone https://github.com/${YOUR_GITHUB_ACCOUNT_NAME}/aws-lc.git
+mkdir aws-lc-build && cd aws-lc-build
+cmake -GNinja \
+    -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+    -DCMAKE_INSTALL_PREFIX=../aws-lc-install \
+    ../aws-lc
+ninja-build run_tests && ninja-build install
+cd ../aws-lc-install/
+ls *
+```
+See [Building.md](https://github.com/awslabs/aws-lc/blob/main/BUILDING.md) for more
+information about required dependencies and build options. If you’re interested in
+getting involved open an Issue to discuss your plan or talk to us on Gitter.
+[Contributing.md](https://github.com/awslabs/aws-lc/blob/main/CONTRIBUTING.md) has
+info for how to specifically make the change and get it reviewed by AWS-LC maintainers.
+If you just want to use AWS-LC see our existing documentation in the public header
+files, if you’re moving your application from OpenSSL see
+[Porting_to_AWS-LC.md](https://github.com/awslabs/aws-lc/blob/main/PORTING_TO_AWSLC.md)
+for more information.
 
 ## Why AWS-LC?
 
-AWS-LCs goal is to maintain a secure libcrypto that is compatible with software and
-applications used at AWS. AWS-LC also serves as the new home for the AWS Cryptography 
-team to publish open source contributions and enhancements that are submitted to 
-other libcrypto projects. 
+AWS-LC's goal is to maintain a secure libcrypto that is compatible with software and
+applications used at AWS. AWS-LC also serves as the new home for the AWS Cryptography
+team to publish open source contributions and enhancements that are submitted to
+other libcrypto projects.
+
+## AWS-LC features
+### API Compatibility
+AWS-LC is compatible with the majority of OpenSSL’s APIs to make it easy to use with
+existing applications. We’re open to discussing adding missing functionality and
+understanding your use case in an Issue or on Gitter.
+
+### Compiler Compatibility
+AWS-LC’s libcrypto is a C99 library and will work with any compiler that supports the
+standard. To build libssl and AWS-LC’s tests you will need a compiler that supports
+C++11. AWS-LC is tested with:
+
+* GCC 4.8.5 to 8.4
+* Clang 7 to 10
+* Visual Studio 2015
+
+For a complete list of tested compilers see
+[tests/ci/Readme.md](https://github.com/awslabs/aws-lc/blob/main/tests/ci/README.md).
+If you use another compiler and would like to make sure we maintain support, please
+open an issue to discuss adding it to our CI.
+
+### OS Compatibility
+
+AWS-LC should work on any system with a supported compiler. AWS-LC has been written to
+support Unix and Windows operating systems. AWS-LC is tested on:
+* Amazon Linux 2
+* Centos 7
+* Ubuntu 18.04 and 20.04
+* Fedora 31
+* Windows Server 10
```

Comment:
Nit: the Docker file uses `FROM mcr.microsoft.com/windows/servercore:ltsc2019`. Is Windows 10 another name of windows server 2019? I saw CodeBuild uses Windows 2019 directly.